deploy/crds/integreatly.org_blobstorages_crd.yaml

```diff
@@ -28,41 +29,12 @@ spec:
         metadata:
           type: object
         spec:
-          properties:
-            secretRef:
-              properties:
-                name:
-                  type: string
-                namespace:
-                  type: string
-              type: object
-            tier:
-              type: string
-            type:
-              type: string
-          required:
-          - type
-          - tier
-          - secretRef
-          type: object
+          $ref: '#/definitions/github.com~1integr8ly~1cloud-resource-operator~1pkg~1apis~1integreatly~1v1alpha1~1types~0ResourceTypeSpec'
```

Comment:
Should we be defining `$ref` in `types`? I am not sure here so I am just asking :smile_cat:

Reply:
no, unfortunately this does allow the CRD to be applied to a cluster. It is because all the types are using the format `type SMTPCredentialSetSpec types.ResourceTypeSpec`

Reply:
Fixed now